pom.xml

```diff
@@ -436,6 +436,11 @@
                         <groupId>thirdparty</groupId>
                         <artifactId>jgrapht-jdk1.6</artifactId>
                     </exclusion>
+                    <exclusion>
+                        <!-- replaced by minlog 1.3 under com.esotericsoftware -->
+                        <groupId>com.esotericsoftware.minlog</groupId>
+                        <artifactId>minlog</artifactId>
+                    </exclusion>
```

Comment:
Does that mean that the new minlog with be a transitive dependency to register a CQ for?